src/regdesk/states/single_result_state.js

```diff
@@ -71,17 +70,15 @@ export class SingleResultState extends RegState {
     this.cancelButton.clearTransitionCallback();
     const ctx = this.badgeCanvas.getContext('2d');
     ctx.clearRect(0, 0, this.badgeCanvas.width, this.badgeCanvas.height);
+
+    this.printButton.clearClickCallback();
   }
 
   /**
-   * Get an age in years between the birthdate and today.
-   * @param {string} birthdate - The raw birthdate string, in YYYY-mm-dd format.
-   * @param {Date} today - The 'today' value to use. Defaults to new Date().
-   * @return {number} - Age in years between the birthdate and today.
+   * Print the label from the latest result.
    */
-  #getAge(birthdate, today = new Date()) {
```

Comment:
Yay! I'm super glad this method was cleaned up. Thank you for your commitment to clean code by reducing copy pastes :D.